public/index.html

```diff
@@ -1,77 +1,90 @@
-<html> 
```

Comment:
Hey sorry about this file, the diff thing hates itself when you change this much white space.